proofs/redexpr.ml

```diff
@@ -25,8 +25,11 @@ open Misctypes
 
 (* call by value normalisation function using the virtual machine *)
 let cbv_vm env sigma c =
-  let ctyp = Retyping.get_type_of env sigma c in
-  Vnorm.cbv_vm env sigma c ctyp
+  if Coq_config.enable_vm then
+    let ctyp = Retyping.get_type_of env sigma c in
+    Vnorm.cbv_vm env sigma c ctyp
+  else
+    compute env sigma c
 
```

Comment:
Notice that AFAIR there are slight differences between the VM and `compute`, so that files that compile with one might not compile with the other.

Reply:
In fact I was afraid of this, Travis won't tell us because it runs with the VM enabled for obvious reasons. Should I use a different set of flags? From my poor understanding of the code compute seemed the closed.

All that I can say is that math-comp, elpi, and iris do compile with the vm disabled.

Reply:
Maybe I should try flocq / compcert.

Reply:
I'm not sure what you guys are trying to test. Obviously, there are some devs that won't compile with the VM off, otherwise we would just get rid of the VM. But that's fine IMHO.

Reply:
Well, I built quite a few devs with the VM off and no problem so far.